docs/Development.md

```
@@ -1,334 +1,240 @@
 
 </details>
 
 <details>
 <summary>Windows</summary>
 
-Use the Minikube IP obtained from `minikube ip` and add a hosts entry:
+Get the IP with `minikube ip`, then add it to `C:\Windows\System32\drivers\etc\hosts`:
 
-```plaintext
-<minikube-ip>    project-331.local
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)